communication/marketing-team/multichannel-communications.md

```diff
@@ -0,0 +1,82 @@
+# Multichannel communications
+
+The goal of this is to provide a clearer way for community members to share and consume Kubernetes information.
+
+## Motive
+Our motive is as follows:
+
+* Provide all Kubernetes contributors with a dependable experience of communication to and from the project
+* Find ways of adding our human touch to potentially overwhelming quantities of information
+* Provide clear expectations on what’s important, urgent, or otherwise
+* Build process like we build Kubernetes, so that our contributors understands it, and it works well at scale
+
+## Outbound and inbound communications
+
+All outbound communications will have a single source of truth in kubernetes-dev@googlegroups.com. 
+
+Justification for this canonical location:
+* Per 2018 and 2019 survey results, it has grown in popularity as the one place to check for contributor news
+* Highlighted in the contributor guide, SIG READMEs, governance documentation, and many other places
+
+All other communication channels will point to that canonical location because we want to:
+
+* avoid having multiple places with the same information
+* avoid confusion on where to get the right information
+* create an easier process for information sharing
+
+## Mapping Type to Tier
+
+We have identified four (4) different tiers of information:
+
+* Action Required
+    * Participation by contributors is required
+    * Example 1: The Contributor Survey
```

Comment:
The story I tell myself is that these examples aren't technically required, but they're urgent for those who care to participate. Reframing that, it might be fair to say: 

```suggestion
    * Participation is critical to the Kubernetes project and has a firm due date 
    * Example 1: Contributor Survey responses are due by $date
```